aiomcache/client.py

```diff
@@ -70,19 +135,16 @@ def _execute_simple_command(self, conn, raw_command):
         return response[:-2]
 
     @asyncio.coroutine
-    def close(self):
-        """Closes the sockets if its open."""
-        yield from self._pool.clear()
-
-    @asyncio.coroutine
-    def _multi_get(self, conn, *keys, with_cas=True):
+    def _multi_get(self, conn, *keys, with_flags=False, with_cas=False):
         # req  - get <key> [<key> ...]\r\n
         # resp - VALUE <key> <flags> <bytes> [<cas unique>]\r\n
         #        <data block>\r\n (if exists)
         #        [...]
         #        END\r\n
+        assert not with_flags, "ascii client does not yet support flags"
```

Comment:
I suggest to use raise exception explicitly, asserts in code may be disabled if one runs python with -O option